app/src/app/components/toolbar/toolbar.component.html

```diff
@@ -10,6 +10,17 @@
                 <button mat-button class="control" [disabled]="!editing || hasRelationships" (click)="delete()"><mat-icon aria-label="delete">delete</mat-icon></button>
             </div>
             <app-object-status *ngIf="hasWorkflow"></app-object-status>
+            <span *ngIf="hasWorkflow" [matTooltip]="!websiteIntegrationService.websiteIntegrationEnabled ? 'external website integration disabled' : (!websiteIntegrationService.currentWebIntegrationStatus.valid ? 'could not find object on att&ck website' : 'view object in att&ck website')">
```

Comment:
ATT&CK should be capitalized. Great use of the tooltips to indicate the disabled state 👍